src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/NavigationControllerFooterView.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+import UIKit
+
+class ENAButtonFooterView: UIView {
+	private(set) var primaryFooterButton: ENAButton!
+	private(set) var secondaryFooterButton: ENAButton!
+
+}
+
+
+extension ENAButtonFooterView {
+	private func setupFooterView() {
+		primaryFooterButton = ENAButton(type: .custom)
+		primaryFooterButton.setTitle("Primary Button", for: .normal)
+		primaryFooterButton.addTarget(self, action: #selector(didTapPrimaryButton), for: .primaryActionTriggered)
+
+		secondaryFooterButton = ENAButton(type: .custom)
+		secondaryFooterButton.setTitle("Secondary Button", for: .normal)
+		secondaryFooterButton.isTransparent = true
+		primaryFooterButton.addTarget(self, action: #selector(didTapSecondaryButton), for: .primaryActionTriggered)
+
+		footerButtonContainerView = UIView(frame: CGRect(x: 0, y: 0, width: 300 + 2 * 16, height: 2 * 50 + 8 + 2 * 16))
+		footerButtonContainerView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
wouldn't the custom frame collapse after setting `.translatesAutoresizingMaskIntoConstraints` to false?

Reply:
that file was committed by mistake. Removed it, thanks for pointing it out. Xcode can really mess up a times when refactoring files.